compute/gpu/reference-content/choosing-gpu-instance-type.mdx

```diff
@@ -0,0 +1,46 @@
+
+## How to choose the right GPU Instance type
+
+Scaleway provides a range of GPU Instance offers, and choosing the right GPU Instance type involves considering several factors to ensure that it meets your performance, budget, and scalability requirements.
+Below, you will find a guide to help you make an informed decision:
+
+* **Workload Requirements:** Identify the nature of your workload. Are you running machine learning, deep learning, high-performance computing (HPC), data analytics, or graphics-intensive applications? Different Instance types are optimized for different types of workloads. For example, the H100 is not designed for graphics rendering. However, the L4 and L40S are. As [stated by Tim Dettmers](https://timdettmers.com/2023/01/30/which-gpu-for-deep-learning/), _“Tensor Cores are most important, followed by memory bandwidth of a GPU, the cache hierarchy, and only then FLOPS of a GPU.”_ For more information, refer to the [NVIDIA GPU portfolio](https://www.nvidia.com/content/dam/en-zz/solutions/data-center/data-center-gpu-portfolio-line-card.pdf).
+* **Performance Requirements:** Evaluate the performance specifications you need, such as the number of GPUs, GPU memory, processing power, and network bandwidth. You need a lot of memory and fast storage for demanding tasks like deep learning.
+* **GPU Type:** Scaleway offers different GPU types, such as various[ add references ? ] NVIDIA GPUs. Each GPU has varying levels of performance, memory, and capabilities. Choose a GPU that aligns with your specific workload requirements.
+* **GPU Memory:** GPU memory bandwidth is an important criterion influencing overall performance. Larger GPU memory is crucial for memory-intensive tasks like deep learning training on large datasets or GPU rendering. Modern GPUs offer specialized data formats designed to optimize deep learning performance. These formats, including Bfloat16, FP8, int8, int4, among others, enable the storage of more data in memory and can result in performance enhancements by a factor of ten. To make an informed decision, it is thus crucial to select the appropriate NVIDIA architecture—options range from Pascal and Ampere to Ada Lovelace and Hopper. Ensuring that the GPU possesses sufficient memory capacity to accommodate your specific workload is essential, preventing any potential memory-related bottlenecks. Equally important is matching the GPU's memory type to the nature of your workload. Keep in mind HBM3 memory surpasses HBM2e, which in turn outperforms HBM2 and GDDR6, making the right memory choice a key consideration.
+* **CPU and RAM:** Do not forget to think about the CPU and system memory. A powerful CPU can be beneficial for tasks that involve preprocessing or post-processing. Sufficient system memory is also crucial to prevent memory-related bottlenecks.
```

Comment:
```suggestion
* **CPU and RAM:**  A powerful CPU can be beneficial for tasks that involve preprocessing or post-processing. Sufficient system memory is also crucial to prevent memory-related bottlenecks.
```